views/newsletters.html

```diff
@@ -29,6 +29,16 @@
       var mailpoet_premium_active = <%= json_encode(premium_plugin_active) %>;
       var mailpoet_transactional_emails_opt_in_notice_dismissed = '<%= transactional_emails_opt_in_notice_dismissed %>';
       var mailpoet_mss_key_invalid = <%= mss_key_invalid ? 'true' : 'false' %>;
+      var mailpoet_product_categories = <%= json_encode(product_categories)  %>;
+      mailpoet_product_categories = mailpoet_product_categories.map(function(category) {
```

Comment:
I would be nicer to do that in PHP so that all mappings happen in one place otherwise it is harder to follow where data comes from.